types/mongoose/test/model.ts

```diff
@@ -230,7 +231,7 @@ MongoModel.deleteOne({_id: '999'}).exec().then(res=>console.log(res.ok));
 MongoModel.deleteMany({_id: '999'}).then(res=>console.log('Success?',!!res.ok, 'deleted count', res.n));
 MongoModel.deleteMany({_id: '999'}).exec().then(res=>console.log(res.ok));
 MongoModel.update({ age: { $gt: 18 } }, { oldEnough: true }, cb);
-MongoModel.update({ name: 'Tobi' }, { ferret: true }, { multi: true,  arrayFilters: [{ element: { $gte: 100 } }] }, cb);
+MongoModel.update({ name: 'Tobi' }, { ferret: true }, { multi: true,  arrayFilters: [{ element: { $gte: 100 } }], overwriteDiscriminatorKey: true }, cb);
```

Comment:
Better not modify the existing line, but add this as a new case. To ensure that without this option, it still works.

Reply:
Done